api/v1/apiserver_types.go

```diff
@@ -78,7 +78,7 @@ func init() {
 type APIServerDeploymentContainer struct {
 	// Name is an enum which identifies the API server Deployment container by name.
 	// Supported values are: calico-apiserver, tigera-queryserver
-	// +kubebuilder:validation:Enum=calico-apiserver;tigera-queryserver
+	// +kubebuilder:validation:Enum=calico-apiserver;tigera-queryserver;calico-l7admssctrl
```

Comment:
Given this name shows up on a user-facing API, can we change the container name to avoid abbreviations?

```
calico-l7-admission-controller
```